README.md

```diff
@@ -32,6 +32,8 @@ To use it, install lxd and initialize it using `lxd init`. When prompted, answer
        root:1000000:1000000000
        <youruserid>:1000000:1000000000
 
+Run `systemctl restart lxd` to have LXD detect your new maps.
```

Comment:
The markdown is missing a two-space indent here. As is, the numbering in the rendered markdown will be 1, 1, 2... instead of 1, 2, 3...